### ISSUE TITLE
feat(background-tasks): prioritising background tasks

### ISSUE THREAD

**Issue by nino-filigran** [LuatixHQ/opencti#4968]
## Use case

All tasks are processed in the same way (last in, last out), which poses a problem when there is a large task in progress but the analyst needs to execute smaller ones manually. The analyst's tasks should always come before the automatic tasks.

## Current Workaround

None. 

## Proposed Solution

Have a way to prioritize certain tasks over others. The solution remain to be defined. 
## If the feature request is approved, would you be willing to submit a PR?

Yes / No (Help can be provided if you need assistance submitting a PR)

**Comment by aqp-tt** [comment]
I fully agree on this issue and in my opinion it should be a priority. When ingesting hundreds of thousand of bundles and processing them takes days, a task done by a user can take up more than a week, which is not acceptable at all. Manual tasks ordered by user should have total priority, or at least the option to set it.

**Comment by nino-filigran** [comment]
In 6.7 we already done some work to improve background tasks, allowing you to have some in parallel. This does not mean that this feature is not needed anymore, but I'd be curious to see if you've noticed some improvements after this fix.
